osu.Game/Screens/Select/BeatmapInfoWedge.cs

```diff
@@ -64,9 +66,11 @@ public BeatmapInfoWedge()
         }
 
         [BackgroundDependencyLoader(true)]
-        private void load([CanBeNull] Bindable<RulesetInfo> parentRuleset)
+        private void load([CanBeNull] Bindable<RulesetInfo> parentRuleset, [CanBeNull] Bindable<IReadOnlyList<Mod>> parentMods)
         {
             ruleset.BindTo(parentRuleset);
+            mods.BindTo(parentMods);
```

Comment:
Wait wtf this doesn't even make any sense if the parent anything is null...

I'm removing this code.